namerd/iface/control-http/src/main/scala/io/buoyant/namerd/iface/HttpControlService.scala

```diff
@@ -106,14 +108,42 @@ object HttpControlService {
       }
   }
 
+  trait NsPathUri {
+    val prefix: String
+
+    def unapply(uri: String): Option[(Ns, Path)] = {
+      if (uri.startsWith(prefix)) {
+        uri.stripPrefix(prefix).split("/").toSeq match {
+          case ns +: path if path.nonEmpty => Some((ns, Path.Utf8(path: _*)))
+          case _ => None
+        }
+      } else {
+        None
+      }
+    }
+  }
+
+  object BindUri extends NsPathUri {
+    val prefix = "/api/1/bind/"
+  }
+
+  object AddrUri extends NsPathUri {
+    val prefix = "/api/1/addr/"
+  }
+
+  object DelegateUri extends NsPathUri {
+    val prefix = "/api/1/delegate/"
+  }
+
```

Comment:
@adleong i think we should consider namespaces as resource trees.  we can create a dtab now (with some backends, at least).  e.g
```
:; namerctl dtab create other/ns - <<EOF
/srv       => /io.l5d.k8s/default/http ;
/host      => /srv ;
/method    => /$/io.buoyant.http.anyMethodPfx/host ;
/http/1.1  => /method ;
EOF
Created other/ns
:; namerctl dtab list
default
other/ns
```

The problem is when we're mixing Dtab Namespaces and Name.Path -- I'd opt to treat the Name.Path as an argument to the Dtab.

@gtcampbell we'll probably need a way of encoding slashes in the k8s store... (maybe just b64 the name and call it a day)